.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -22,7 +22,7 @@ This checklist is for your information.
 - [ ] Give a meaningful name to your PR, as it may end up being used in the release notes.
 - [ ] Your code is flake8 compliant.
 - [ ] Your code is python 3.6 compliant (specific python >3.6 syntax is currently not accepted).
-- [ ] If this is a new feature and not a bug fix, you've included the proper documentation in the ReadTheDocs documentation folder. https://github.com/DefectDojo/Documentation/tree/master/docs or provide feature documentation in the PR.
+- [ ] If this is a new feature and not a bug fix, you've included the proper documentation in the documentation folder. https://github.com/DefectDojo/django-DefectDojo/tree/dev/docs or provide feature documentation in the PR.
```

Comment:
I'd go with just `If this is a new feature and not a bug fix, you've included the proper documentation in the docs at https://github.com/DefectDojo/django-DefectDojo/tree/dev/docs as part of this PR`.
`